Doc/library/asyncio-task.rst

```diff
@@ -109,34 +109,53 @@ To actually run a coroutine asyncio provides three main mechanisms:
 Awaitables
 ==========
 
-We say that an object is an *awaitable* object if it can be used
-in an :keyword:`await` expression.
+We say that an object is an **awaitable** object if it can be used
+in an :keyword:`await` expression.  Many asyncio APIs are designed to
+accept awaitables.
 
+There are three main types of *awaitable* objects:
+**coroutines**, **Tasks**, and **Futures**.
 
-.. rubric:: Coroutines and Tasks
 
-Python coroutines are *awaitables*::
+.. rubric:: Coroutines
+
+Python coroutines are *awaitables* and therefore can be awaited from
+other coroutines::
 
     async def nested():
         return 42
 
     async def main():
-        # Will print "42":
-        print(await nested())
+        # Nothing happens if we just call "nested()":
+        nested()  # a coroutine object is created but it is
+                  # not awaited and not wrapped into a Task.
 
-*Tasks* are used to schedule coroutines *concurrently*.
-See the previous :ref:`section <coroutine>` for an introduction
-to coroutines and tasks.
+        # Let's do it differently now and await it:
+        print(await nested())  # will print "42".
 
 Note that in this documentation the term "coroutine" can be used for
 two closely related concepts:
 
 * a *coroutine function*: an :keyword:`async def` function;
 
-* a *coroutine object*: object returned by calling a
```

Comment:
Let's put this note in a note box so it catches the reader's eye. There's a fair amount of confusion between the two.

Reply:
Hum, using a box will break the flow of text (and it's not a lot of text in this section). I prefer the reader to quickly read the section quickly without too much distraction. If they miss a thing or two they'll probably reread it later.

Reply:
As a compromise let's use ".. important:" directive which is less visually disruptive but still highlights the paragraph.